packages/@cdktf/cli-core/src/lib/models/terraform-cli.ts

```diff
@@ -215,6 +225,16 @@ export class TerraformCli implements Terraform {
     } = opts;
     const options = ["plan", "-input=false"];
 
+    const generatedConfigFile = path.join(
+      this.workdir,
+      "generated_resources.tf"
+    );
+    if (fs.existsSync(generatedConfigFile)) {
+      fs.remove(generatedConfigFile);
+    }
+    if (this.hasImports) {
+      options.push("-generate-config-out=generated_resources.tf");
```

Comment:
nit: We could pull `generated_resources.tf` into a constant and use it here and further down below.
(only a nit since they are in the same file here 😄)

Reply:
👍